Sources/SimpleCurrencyConverter/ExchangeRatesNetworkManager.swift

```diff
@@ -27,8 +40,8 @@ final class ExchangeRatesNetworkManager {
             }
             
             var request = URLRequest(url: modifiedURL)
-            request.httpMethod = "GET"
-            request.setValue(apiKey, forHTTPHeaderField: "apikey")
+            request.httpMethod = Keys.httpMethod
```

Comment:
Actually you could have left `"GET"` string here. Now the name became vague and doesn't indicate what kind of method you are using. Later if you add "POST" you'll have to change the name.
So either leave `"GET"` or create an enum with that has "GET" in the name